src/kudu/consensus/raft_consensus.cc

```diff
@@ -579,6 +589,12 @@ Status RaftConsensus::StartElection(ElectionMode mode, ElectionContext context)
     LockGuard l(lock_);
     RETURN_NOT_OK(CheckRunningUnlocked());
 
+    if (!persistent_vars_->is_start_election_allowed()) {
+      LOG_WITH_PREFIX_UNLOCKED(INFO) <<
```

Comment:
this is going to flood the logs when we have disabled start election and the failure detector is still not turned off.